### DISCUSSION TITLE
Not send stack on exceptions

### DISCUSSION THREAD

**Discussion by xarielah** [typestack/class-validator#1889] [Questions & Answers]
Hi,

I was wondering if thats possible to not send the stack when ever an exceptions is thrown?

![image](https://user-images.githubusercontent.com/84734223/211209179-b9f38770-d78c-457f-b8ad-e7d23ba6a5b0.png)

**Comment by braaar** [comment]
It doesn't seem like this is an error thrown by class-validator. Could you provide a reproduction?

**Reply by xarielah** [reply]
It is, just with a custom text within the error thrown for example:

throw new NotFoundExceptions("...text");

**Reply by braaar** [reply]
class-validator doesn't have anything to do with any HTTP status codes, which we can see in your screenshot. Can you try to use try/catch in your program to isolate the error that class-validator itself is throwing? 

Also keep in mind that class-validator may not throw any errors at all if you are using `validate` (there are ways to get class-validator to `throw` but `validate` returns validation errors)

**Reply by xarielah** [reply]
> class-validator doesn't have anything to do with any HTTP status codes, which we can see in your screenshot. Can you try to use try/catch in your program to isolate the error that class-validator itself is throwing? 
> 
> 
> 
> Also keep in mind that class-validator may not throw any errors at all if you are using `validate` (there are ways to get class-validator to `throw` but `validate` returns validation errors) 

After saying that, I'm using next-api-decorators, stack property might be added from there I guess. Btw what was that last thing you said about validate? Is that a keyword I can use in typescript? Can you direct me to docs or explanation of that please?

**Reply by braaar** [reply]
[`validate`](https://github.com/typestack/class-validator#usage) is the function you call to perform validation with class-validator. If you are using class-validator wrapped by some other package the actual call may be abstracted away. There is also the `validateOrReject` method which will `throw` an error if the object doesn't pass validation.
